src/components/common/Stepper.vue

```diff
@@ -1,12 +1,17 @@
 <template>
-  <v-container id="step-buttons-container" class="list-item justify-space-around py-8">
+  <v-container id="step-buttons-container">
     <template v-for="(step, index) in steps">
-      <div class="text-center align-self-start" v-bind:key="index">
-        <v-btn :id=step.id outlined fab color="primary" :to=step.to :disabled=step.disabled>
-          <v-icon>{{step.icon}}</v-icon>
-        </v-btn>
-        <v-icon size="30" color="green darken-1" v-show=step.valid>mdi-check-circle</v-icon>
-        <div class="pre-line">{{step.text}}</div>
+      <div class="step" v-bind:key="index">
```

Comment:
Not sure you 'have' too put v-bind before the `:key` but thats up to you!

Reply:
You're right, not required. But hey it's a "prototype".

Reply:
Ok, I didn't see this earlier and remove it...

Reply:
right on!